### PR TITLE
PPTT-3045 Replaced all CPP shortsuts with full term and links to glossary

### PR DESCRIPTION
**Pull request by RomanSek**
[Preview](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/search.html?q=Collecting+%26+Processing+Pipeline&check_keywords=yes&area=default)

### CODE REVIEW

**Comment by RomanSek** [comment]
I changed all instances of CPP to links to glossary term, but in some cases this documentation looks more complicated then it needs to be.
1. [Collecting & Processing Pipeline with custom domain](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/tag_manager/content_security_policy.html?highlight=Collecting%20%26%20Processing%20Pipeline#collecting-processing-pipeline-with-custom-domain) maybe it would be better to name it "PPAS with custom domain" or "Tracking with custom domain"?
2. [Collecting & Processing Pipeline Tracking HTTP API](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/index.html) looks very complicated when compared with "Tracking JavaScript API"

3. HTTP API for CPP are mentioning it only in title and description but it's not mentioned in OpenAPI documentation.
[Collecting & Processing Pipeline Tracking HTTP API](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/data_collection/api/http_api.html?highlight=Collecting%20%26%20Processing%20Pipeline)

[Collecting & Processing Pipeline debugger API](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/data_collection/api/tracker_debugger_api.html?highlight=Collecting%20%26%20Processing%20Pipeline)

[Collecting & Processing Pipeline Settings API](https://developers.piwik.pro/en/feature-pptt-3045-replace-cpp-shortcut/platform/authorized_api/tracker_settings/tracker_settings_api.html?highlight=Collecting%20%26%20Processing%20Pipeline)

File: tag_manager/content_security_policy.rst [hunk 1]
```diff
@@ -150,10 +150,10 @@ When using a data subject request widget, you need to add a nonce attribute to i
   </div>
 
 
-CPP with custom domain
----------------
+Collecting & Processing Pipeline with custom domain
```
Comment (azarecki) [inline tag_manager/content_security_policy.rst:153]: Let's got with Tracking with custom domain

File: data_collection/api/http_api.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .. _data-collection-tracking-api-http:
 
-CPP Tracking HTTP API
-=====================
-CPP Tracking HTTP API allows sending to analytics information about Visitors page views, events and visits.
+Collecting & Processing Pipeline Tracking HTTP API
```
Comment (azarecki) [inline data_collection/api/http_api.rst:3]: Tracking HTTP API should be enough

**Review by azarecki — COMMENTED** [review]
Can you change the Open API? It should be the same.

**Review by azarecki — APPROVED** [review]

**Review by wronan — APPROVED** [review]

**Pull request merged**